bin/cisc_to_adie_conversion.py

```diff
@@ -0,0 +1,82 @@
+# Script to convert CISC ID to ADIE ID in directory and file names 
+# This is only tested on directories/file£s with a BIDS naming convention
+# The way the script searches for the sub- ID is dependent on the characters before and after,
+# which is specific to BIDS - be aware  
+
+import pandas as pd
+import os 
+import re
+import shutil
+import sys, os
+
+'''
+# Construct path of converter .txt file such that it's relative to the user running the script
+# Get parent directory of this script       
+pathname = os.path.dirname(sys.argv[0])  
+# Get the level above that (i.e. the critchley_adie project dir)      
+project_path = os.path.split(pathname)[0]
+print('project_path=',project_path)
+'''
+
+#TODO: make this path relative to the user 
+#Import adie/cisc conversion txt file and store as dataframe
+txt=("/Volumes/cisc2/projects/critchley_adie/BIDS_data/sourcedata/adie_idconvert.txt")
+
+# convert this to dictonary, where key = CISC and val = ADIE 
+rename = {}
+with open(txt) as f:
+    for line in f:
+        (key,val)=line.split()
+        rename[str(key)] = str(val)
+
+
+# USE OS.WALK
+
+def subconvert(p):
+# P should = the path of the directory above the sub- dirs 
```

Comment:
Look up the standard python function documentation format. Have a look at any function I wrote.